README.md

```diff
@@ -108,5 +111,5 @@ Your answer here.
 How is `git pull` related to `git fetch`?
 
 ```text
-Your answer here.
+Git pull will automatically merge all of your branches by fetching them. Git fetch gathers all of your commits and stores them locally.
```

Comment:
:+1:

This sounds close enough.  Which branches get merged can be controlled by git configuration.  I recommend you read our recommendation in the solution branch.